apps/foundation-demo/src/app/checkout/components/payment/billing-summary/billing-summary.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="billing-summary">
+  <div class="billing-summary__title">Billing Address</div>
+  <address-summary *ngIf="(billingAddress && !billingAddressIsShippingAddress)" [address]="billingAddress"></address-summary>
+  <div class="billing-summary__billing-address-is-shipping-address" *ngIf="billingAddressIsShippingAddress">Billing address is shipping address</div>
```

Comment:
This classname is silly. Let's generalize it a bit more.

Reply:
renamed to billing-summary__note